server/data/data-api/src/main/java/org/apache/james/user/api/UsersRepositoryManagementMBean.java

```diff
@@ -91,55 +89,6 @@
      */
     void setPassword(String userName, String password) throws Exception;
 
-    /**
-     * Removes a user's alias which terminates local mail forwarding
-     * 
-     * 
-     * @param userName
-     *            The name of the user whose alias is unset
-     * @throws UsersRepositoryException
-     *             if error
-     */
-    @Deprecated
-    void unsetAlias(Username userName) throws Exception;
```

Comment:
as long as we will support JMX, all MBean classes should be considered a public API of James.
Removing and/or changing them should be considered an api breakage.

Reply:
We can still keep the endpoint (akka the API methods), and throw unsupported exceptions as the previous code always ended up throwing IllegalArgumentExceptions....

Would that be acceptable to you?

Reply:
See https://github.com/linagora/james-project/pull/2924 for announcing that breaking change.